src/pages/Account/Login.scss

```diff
@@ -18,13 +26,6 @@
     margin-bottom: $spacing--xs;
   }
 
-  &__quick-login-icon {
-    &:hover {
-      cursor: pointer;
-      opacity: 0.8;
-    }
-  }
```

Comment:
Why were these styles removed from here? Should just be commented out + an `@debt` comment about this being an environment specific hack at most. Though would likely be find to just leave them in place presumably?

Reply:
They just weren't needed anymore. The point of this PR was to fix some styles and make the whole okta box clickable rather than the icon, so having styles for the quick login icon's hover is not right. I can comment it out and write a comment though.